packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -218,8 +227,7 @@ contract OptimisticGovernor is Module, Lockable {
     function executeProposal(
         uint256 _proposalId,
         Transaction[] memory _transactions,
-        uint32 _originalTime,
-        SkinnyOptimisticOracleInterface.Request memory _request
+        uint32 _originalTime
```

Comment:
Something we may want to consider in a follow-up is just storing this timestamp so the user doesn't have to resubmit it, as it may be annoying for the user or the tooling to determine the block timestamp of the original proposal.